pkg/exchange/bitget/types.go

```diff
@@ -425,3 +427,63 @@ type AccountEvent struct {
 	actionType ActionType
 	instId     string
 }
+
+type Trade struct {
+	// Latest filled price
+	FillPrice fixedpoint.Value `json:"fillPrice"`
+	TradeId   types.StrInt64   `json:"tradeId"`
+	// Number of latest filled orders
+	BaseVolume fixedpoint.Value           `json:"baseVolume"`
+	FillTime   types.MillisecondTimestamp `json:"fillTime"`
+	// Transaction fee of the latest transaction, negative value
+	FillFee fixedpoint.Value `json:"fillFee"`
+	// Currency of transaction fee of the latest transaction
+	FillFeeCoin string `json:"fillFeeCoin"`
+	// Direction of liquidity of the latest transaction
+	TradeScope string `json:"tradeScope"`
+}
+
+type Order struct {
+	Trade
+
+	InstId string `json:"instId"`
+	// OrderId are always numeric. It's confirmed with official customer service. https://t.me/bitgetOpenapi/24172
+	OrderId       types.StrInt64 `json:"orderId"`
+	ClientOrderId string         `json:"clientOid"`
+	// Size is base coin when orderType=limit; quote coin when orderType=market
+	Size fixedpoint.Value `json:"size"`
+	// Buy amount, returned when buying at market price
+	Notional      fixedpoint.Value           `json:"notional"`
+	OrderType     v2.OrderType               `json:"orderType"`
+	Force         v2.OrderForce              `json:"force"`
+	Side          v2.SideType                `json:"side"`
+	AccBaseVolume fixedpoint.Value           `json:"accBaseVolume"`
+	PriceAvg      fixedpoint.Value           `json:"priceAvg"`
+	Status        v2.OrderStatus             `json:"status"`
+	CTime         types.MillisecondTimestamp `json:"cTime"`
```

Comment:
for field names, you don't have to use the same abbreviation, full qualified name is always better

Reply:
I also updated another field, but not excluding v1.